frontend/src/components/custom-tools/prompt-card/PromptCard.jsx

```diff
@@ -318,7 +318,7 @@ function PromptCard({
 
     if (runAllLLM) {
       let selectedProfiles = llmProfiles;
-      if (!coverAllDoc && selectedLlmProfiles?.length > 0) {
+      if (selectedLlmProfiles?.length > 0) {
```

Comment:
@jagadeeswaran-zipstack NIT: This can be simplified by just checking the length of the `selectedLlmProfiles`.
```suggestion
      if (selectedLlmProfiles?.length) {
```